pkg/ddevapp/utils.go

```diff
@@ -228,7 +231,7 @@ func CreateGitIgnore(targetDir string, ignores ...string) error {
 
 		// If we sigFound the file and did not find the signature in .ddev/.gitignore, warn about it.
 		if !sigFound {
-			util.Warning("User-managed %s will not be managed/overwritten by ddev", gitIgnoreFilePath)
+			util.WarningOnce("User-managed %s will not be managed/overwritten by ddev", gitIgnoreFilePath)
```

Comment:
I wanted to do this a long time ago. I probably mentioned seeing several identical warnings on `ddev start`, but I never fixed it. Now it's time to do it.